File: Baekjoon/구현/1748.cpp
// https://www.acmicpc.net/problem/1748

#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int n;
    cin >> n;

    int digits = 0;
    for (int i = 1; i <= n; i *= 10) {
        digits += n - i + 1;
    }

    cout << digits << endl;

    return 0;
}

File: Baekjoon/구현/1748.py
# https://www.acmicpc.net/problem/1748

n = input()
cnt = 0

for i in range(1, len(n)):
    cnt += 9 * 10 ** (i - 1) * i

cnt += (int(n) - 10 ** (len(n) - 1) + 1) * len(n)
print(cnt)
